src/sbvr-api/permissions.ts

```diff
@@ -810,7 +813,7 @@ const rewriteRelationship = memoizeWeak(
 						_.set(
```

Comment:
I think this can also have
```ts
if (collapsedPermissionFilters == null) {
	// If we have full access already then there's no need to check for/rewrite based on `canAccess`
	return;
}